Automattic/wp-calypso
# Stats: Updating summary page padding for small screens

Related to #72755

## Proposed Changes

* apply different padding for rows of items on the summary pages

## Testing Instructions



* open the live branch and go to summary pages by clicking on `View all` or `View details` on different stat modules cards
* resize the browser to screen smaller than 780px and 660px
* verify that the rows of items are aligned with the headers
* check rows with children like `Clicks`

| Before | After |
| --- | --- |
| ![SCR-20230222-e5i](https://user-images.githubusercontent.com/112354940/220459569-46ba5346-ca1a-49dd-8b07-268ec746b97e.png) | ![SCR-20230222-e4a](https://user-images.githubusercontent.com/112354940/220459611-729495af-4c50-4e39-92b5-b0f9db74bf79.png) |

## Pre-merge Checklist
